### ISSUE TITLE
Polishing failures on certain datasets

### ISSUE THREAD

**Issue by lcoombe** [bcgsc/goldrush-edit#4]
I was running GoldRush on a subset of the human HG01243 dataset (chr15 only), and saw a reproducible error with GoldRush-Edit:
```
[2022-08-24 10:35:54]ESC[32m[INFO] ESC[0mserve: Targeted BF builder done!
[2022-08-24 10:35:55]ESC[32m[INFO] ESC[0mPolisher done
Traceback (most recent call last):
  File "/projects/btl/lcoombe/miniconda3/envs/goldrush_paper/bin/goldrush-edit-polish-batch", line 150, in <module>
[2022-08-24 10:35:55]ESC[31m[ERROR] ESC[0mntedit: error: `/dev/shm/mSXleaZWSeSMNDLbfPmCVg-targeted_bfs/3497-k24.bf': No such file or directory
make[1]: *** [/dev/shm/mSXleaZWSeSMNDLbfPmCVg-3497/batch.ntedited.fa] Error 1

    cleanup_bfs(args.bfs_dir, args.b)
  File "/projects/btl/lcoombe/miniconda3/envs/goldrush_paper/bin/goldrush-edit-polish-batch", line 102, in cleanup_bfs
Traceback (most recent call last):
  File "/projects/btl/lcoombe/miniconda3/envs/goldrush_paper/bin/goldrush-edit-polish-batch", line 144, in <module>
    os.remove(join(bfs_dirpath, bf))
FileNotFoundError: [Errno 2] No such file or directory: '/dev/shm/mSXleaZWSeSMNDLbfPmCVg-targeted_bfs/3376-k32.bf'
    polishing_stdout, polishing_stderr = run_polishing(
  File "/projects/btl/lcoombe/miniconda3/envs/goldrush_paper/bin/goldrush-edit-polish-batch", line 95, in run_polishing
    raise e
  File "/projects/btl/lcoombe/miniconda3/envs/goldrush_paper/bin/goldrush-edit-polish-batch", line 77, in run_polishing
    sealer_protocol_process = sp.run(
  File "/projects/btl/lcoombe/miniconda3/envs/goldrush_paper/lib/python3.9/subprocess.py", line 528, in run
[2022-08-24 10:35:55]ESC[31m[ERROR] ESC[0mmake[1]: *** No rule to make target `/dev/shm/mSXleaZWSeSMNDLbfPmCVg-3073/batch.ntedited.prepd.sealer_scaffold.upper.fa'.  Stop.
```

We've had plenty of successful GoldRush-Edit runs with this release and installation previously, so it for some reason seems to be related to this particular golden path and/or reads?

I was able to reproduce this by just specifying a single goldtig for polishing.

If I instead used the second from most recent commit (https://github.com/bcgsc/goldrush-edit/commit/4ca8f17d5d2f4932060da705c7e26f1ba8f2d3d8), the polishing finishes just fine, so looks to possibly be related to something changed in the most recent commit, with cleaning up processes?

**Comment by lcoombe** [comment]
Updating that I now don't believe this is limited to smaller datasets. I've seen the behaviour on a human-sized full polishing. The failure sometimes causes the full GoldRush pipeline to fail, but not always in my experience. If it doesn't cause a full failure, the consequence could be a run where some goldtigs are missing from the post-polishing file

**Comment by lcoombe** [comment]
Resolved with #5
